Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -33,11 +33,18 @@ public typealias PurchaseResultData = (transaction: StoreTransaction?,
 /**
  Completion block for ``Purchases/purchase(product:completion:)``
  */
+#if swift(>=5.5) && canImport(_Concurrency)
```

Comment:
I _think_ the swift concurrency import would work?

Reply:
I think since this is a compiler conditional, `canImport` only checks if the version of Xcode you're compiling with supports concurrency.

And since `@MainActor` is supposedly back ported to macOS 10/11, and the current version of Xcode supports concurrency, this means we would end up always compiling the first branch.